docs/src/css/custom.css

```
@@ -313,3 +313,11 @@
 	margin-left: auto;
 	margin-right: auto;
 }
+
+.markdown video {
+	display: block;
+	width: 100%;
+	height: auto;
+	border-radius: 80.5rempx;
```

Stylelint (17.7.0)

[error] Unknown unit "rempx" (unit-no-unknown)

(unit-no-unknown)